api/src/utils/get-milliseconds.ts

```diff
@@ -0,0 +1,14 @@
+import ms from 'ms';
+
+/**
+ * Safely parse human readable time format into milliseconds
+ */
+function getMilliseconds<T>(value: unknown, fallback?: T): number | T;
+function getMilliseconds(value: unknown, fallback = undefined): number | undefined {
+	if ((typeof value !== 'string' && typeof value !== 'number') || value === '') {
+		return fallback;
+	}
+	return ms(String(value)) ?? fallback;
+}
+
+export default getMilliseconds;
```

Comment:
(This is a tangent here, so I've opted to approve the PR as it works great)

Do we currently have a standard/stance on export default, maybe in the context of util functions specifically? 

It seems like a majority of util functions are not using export default. There are still a handful that are still using export default (`applyQuery`, `getASTFromQuery`, `getDefaultValue` etc), but I'm also wondering could it be because they were written quite a long time ago. Just curious whether this is something that we'd like to standardized in terms of code style. 😃

Reply:
> Do we currently have a standard/stance on export default, maybe in the context of util functions specifically?

I was actually wondering the same thing 😄 I thought that a default export is appropriate here, since it will almost certainly remain with this single function.
But let's keep this question in mind for the future 👍

Reply:
> 1. Makes it easier to find globally. Eg when you do a search-in-workspace in VS Code, you can't assume that you've found all results, as the name can be different per import

Never looked at it from this perspective. Very good point 👍